### ISSUE TITLE
Ultrawide 21:9 Support Does Not Work!

### ISSUE THREAD

**Issue by Kurasami** [TitansBane/Elysium-Remastered#130]
Followed the instructions **perfectly** on a fully fresh install of Skyrim and Elysium.
I'm running a 34" 21:9 3440x1440 monitor.

![Screenshot_1](https://github.com/TitansBane/Elysium-Remastered/assets/77334270/39ec6565-9320-4ea7-a126-96c833bffff4)

Enabled all the 44 Widescreen Support mods and dragged the fix underneath SkyUI in the load order.

The game loads but the screen is completely black. I can hear my character walking around, swinging weapon etc. but I cannot see a single thing. I can also see the UI but everything else is totally black.

Can someone assist please?

**Comment by Kurasami** [comment]
Ok so after I enabled the above, the game ran once to that black screen issue.

Now it won't even start. It just tries to load. Has the unlock button, briefly goes black and then MO2 reloads the mods. The ultrawide fix instructions definitely do not work.

**Comment by Kurasami** [comment]
So I fixed it. I think the mistake was I right clicked line 44 and clicked "Enable All". So likely I also enabled conflicting mods.

I re-ran Wabberjack. It (without overwite) and it fixed about 60 files. Then I went back to MO2 and individually clicked each of the widescreen support mods in 44. Loaded the game and it works fine. The UI looks great and now I can finally actually play properly without it chopping off 33% of the UI.

Really looking forward to playing properly now.

Thanks and I hope this helps someone.
